ocs_ci/ocs/resources/cloud_manager.py

```
@@ -119,6 +121,16 @@ def __init__(self):
         except ClusterNotInSTSModeException:
             setattr(self, "aws_sts_client", None)
 
+        # set the client for Azure STS enabled cluster
+        try:
+            setattr(
+                self,
+                "azure_sts_client",
+                cloud_map["AZURE_STS"](full_auth_dict=cred_dict),
+            )
```

Ruff (0.15.14)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)